media.css

```diff
@@ -0,0 +1,1400 @@
+    text-align: center;
+  }
+
+  .video-content__description {
+    width: 100%;
+    font-size: 14px;
+    line-height: 17px;
+    margin-bottom: 41px;
+    color: #fff;
+    text-align: center;
+  }
+
+  .catalog-item__title {
+  	padding-left: 0;
+  }
+
+  .catalog-item__description {
+  	padding-left: 0;
+  }
+
+  .stories-content {
+    width: 100%;
+    padding-bottom: 110px;
+    background-color: #000;
+    color: #fff;
+    margin-top: -5px;
+  }
+
+  .stories-content__h4 {
+    font-size: 20px;
+    line-height: 19px;
+    font-weight: bold;
+    text-align: center;
+    text-transform: uppercase;
+    margin-bottom: 25px;
+  }
+
+  .stories-items {
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+  }
+
+  .stories-items__item {
+    width: 87%;
+  }
+
+  .stories-item__image {
+    display: block;
+    width: 100%;
+    margin: 0 auto;
+    margin-bottom: 29px;
+  }
+
+  .stories-item__title {
+    font-size: 20px;
+    line-height: 19px;
+    font-weight: bold;
+    margin-bottom: 9px;
+  }
+
+  .stories-item__description {
+    font-size: 14px;
+    line-height: 17px;
+    margin-bottom: 29px;
+  }
+
+  .stories-item__description-extended {
+    width: 100%;
+    display: grid;
+    grid-template-columns: 0.75fr 0.5fr 1fr;
+  }
+
+  .hidden-1024 {
+    display: flex;
+    align-items: center;
+  }
+
+  .stories-item__date {
+    font-size: 15px;
+    line-height: 14px;
+    font-weight: bold;
+    color: #d1d1d1;
+    opacity: 1;
+    text-align: right;
+  }
+
+  .gallery {
+    display: block;
+    height: 17%;
+    background: linear-gradient(2.6deg, #fff 50%, #000 51%);
+    margin-top: -5px;
+    width: 100%;
+  }
+
+  .gallery__title {
+    font-size: 25px;
+    line-height: 24px;
+    color: #e8e8e8;
+    opacity: 1;
+    text-align: center;
+    text-transform: uppercase;
+    font-weight: bold;
+    margin-bottom: 17px;
+    margin-right: 0;
+  }
+
+  .gallery-content {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: 3;
+    margin-left: 20px;
+    margin-right: 20px;
+  }
+
+  .gallery-content__item {
+    display: block;
+    margin-bottom: -4px;
+  }
+
+  .gallery__photo {
+    width: 100%;
+  }
+
+  .hidden-1024 {
+    display: block;
+  }
+
+  .footer {
+    width: 100%;
+    min-width: 319px;
+  }
+
+  .footer__content {
+    max-width: 88%;
+    padding-top: 40px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .footer__column {
+    text-align: center;
+    margin-bottom: 52px;
+  }
+
+  .footer__column-title,
+  .footer-column__nav-link {
+    font-size: 15px;
+    line-height: 14px;
+  }
+
+  .copyright {
+    padding-top: 30px;
+    margin-top: -52px;
+  }
+
+  .footer__button {
+    width: 150px;
+    height: 37px;
+    margin-top: 29px;
+    margin-bottom: 104px;
+    margin-right: auto;
+    margin-left: auto;
+  }
+
+  .socials {
+  	padding-right: 0;
+  	display: flex;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)